account_financial_report/report/general_ledger.py

```diff
@@ -365,11 +360,10 @@ def _get_period_domain(
         only_posted_moves,
         date_to,
         date_from,
-        analytic_tag_ids,
         cost_center_ids,
     ):
         domain = [
-            ("display_type", "=", False),
+            ("display_type", "not in", ["line_note", "line_section"]),
```

Comment:
Sorry, what is the reason for this change? (appears in several places)

Reply:
v16 has changes in this field usage, and has more values than old versions

Reply:
Ok, sorry, I didn't know it, thank you very much.